lib/constantcontact/services/email_marketing_service.rb

```diff
@@ -51,6 +51,21 @@ def get_campaign(campaign_id)
           Components::Campaign.create(JSON.parse(response.body))
         end
 
+        # Get campaign email content for a specific campaign
+        # @param [Integer] campaign_id - Valid campaign id
+        # @return [Campaign]
+        def get_campaign_preview(campaign_id)
+          url = Util::Config.get('endpoints.base_url') +
+                sprintf(Util::Config.get('endpoints.campaign_preview'), campaign_id)
+          url = build_url(url)
+          response = RestClient.get(url, get_headers())
+          data = JSON.parse(response.body)
+          campaign = Components::Campaign.create(data)
```

Comment:
My only question remaining is, does this work for you guys? This isn't technically a campaign but creating a new object to wrap this response didn't make sense to me either. I'll leave that decision to you all. Just food for thought. :)